### PR TITLE
resolve #25 editing form fields, interpolation and default value

### LINKED ISSUE #25: The addresses of the students are not properly shown and modified
The addresses of the students (the legal and the customizable one) are not properly shown in the interface. When trying to change the address, the modifications are not taken into account.

### PR DESCRIPTION
**Pull request by mathieuzen**

### CODE REVIEW

**Comment by mathieuzen** [comment]
last commit is to revert the commit that was made for PR #31 in order to differentiate each issue and its fix - let me know if it has to be done differently

File: models/organization.py [hunk 1]
```diff
@@ -48,6 +48,7 @@ class Organization(SerializableModel):
     city = models.CharField(max_length=255, blank=True, null=True)
     country = models.ForeignKey('reference.Country', blank=True, null=True)
     cohort = models.ForeignKey('internship.Cohort', null=False, on_delete=models.CASCADE)
+    address = [location, postal_code, city, country]
```
Comment (htmfilho) [inline models/organization.py]: I didn't get this change. Is it related to the problem ?

File: models/organization.py [hunk 2]
```diff
@@ -115,7 +116,7 @@ def sort_organizations(organizations):
         if organization is not None:
             number_ref.append(organization.reference)
     if number_ref:
-        number_ref = sorted(number_ref, key=int)
+        number_ref = sorted(number_ref, key=lambda ref: int(ref) if ref.isdigit() else ref)
```
Comment (htmfilho) [inline models/organization.py]: I didn't get this change either. Is it related to the problem ?

File: models/organization.py (hunk 1 above)
Comment (mathieuzen) [reply models/organization.py]: An error was preventing me to enter the information page for the student as the template was looking for organization.address.location, organization.address.postal_code, etc. and address was undefined. 

Therefore, the change was either to make 4x in the template or once in the model.

File: models/organization.py (hunk 2 above)
Comment (mathieuzen) [reply models/organization.py]: In the data I use in local I didn't know if ref was a number or a string so I created a new organization with a string as reference. This error was preventing me to access the information page too.

We should probably enforce admin to use int when encoding reference for an organization and drop this change.

File: models/organization.py (hunk 1 above)
Comment (htmfilho) [reply models/organization.py]: Ok, but I can't find where you are using the variable `address`.

Comment (mathieuzen) [reply models/organization.py]: https://github.com/uclouvain/osis-internship/blob/17242086d0a200a1ca3c6403d0709d1d3f4aabf5/templates/student.html#L144-L149

But it has been changed lately so we should indeed not consider this line.

`address = [location, postal_code, city, country]`

**Pull request closed without merging**